src/renderer/containers/App/handleUploadJobUpdates.ts

```diff
@@ -73,6 +73,7 @@ function handleFSSMultifileJobUpdate(job: FSSUpload, dispatch: Dispatch<any>) {
       })
     );
   }
+  // TODO?: Maybe raise error here if there is no subfiles?
```

Comment:
Good point. When FSS first creates an upload tracking job, there is no `serviceFields.subfiles` property. While I'm not sure exactly when in the upload FUA will get the first job update, it's possible that the very first update of a valid multifile upload will have no subfiles.

Reply:
Yeah, good point. There should be at least one place around here where the `subfiles` property is used to do some math - let's make sure there's a check, wherever that happens, for the existence of the `subfiles` property to begin with.

Reply:
We look at it [here ](https://github.com/aics-int/aics-file-upload-app/blob/734c6b07a5baa3e7c53f99d177ae067045f1074b/src/renderer/containers/App/handleUploadJobUpdates.ts#L64)